### PR TITLE
added golang session data

### PR DESCRIPTION
**Pull request by atharvakadlag**

### CODE REVIEW

**Comment by atharvakadlag** [comment]
Some checks failed, waiting for @PuruVJ to review

File: src/routes/edith.svelte [hunk 1]
```diff
@@ -41,6 +41,47 @@
 			<!-- TODO: put everything inside a 'for' loop and have an array of events
            in JS -->
 			<div>
+				<Card actions id="golang-crash-course">
+					<span slot="content"
+						>GoLang Crash Course/span>
```
Comment (PuruVJ) [inline src/routes/edith.svelte:46]: should be </span>

Comment (PuruVJ) [reply src/routes/edith.svelte:46]: Use the Svelte official extension na 😅

Comment (atharvakadlag) [reply src/routes/edith.svelte:46]: 💩 sorry

Comment (atharvakadlag) [reply src/routes/edith.svelte:46]: thought it wasnt that big a change so wouldnt require, I added it now.

**Comment by PuruVJ** [comment]
Built successfully 👍
Feel free to merge

**Pull request merged**